kindelia_core/genesis.kdl

```diff
@@ -20,7 +20,7 @@ ctr {TG x0 x1 x2 x3 x4 x5 x6 x7 x8 x9 x10 x11 x12 x13 x14 x15}
 // An if-then-else statement
 fun (If cond t f) {
   (If #0 ~ f) = f
-  (If  ~ t ~) = t
+  (If #1 t ~) = t
```

Comment:
I don't think making `If` incomplete is a good idea.

Reply:
When I was writing `nonce.kdl`, it took me like 2 hours to figure out why it was always falling in the true case, even in obviously false cases.

It was because I was comparing `(== #5 {T1 #6})` and the computation was getting stuck. The way it was, even stuck terms were being evaluated to true, which caused weird unexpected behaviour. I'm even in favor of adding rules like `(If {True} ...)` and `(If {False} ...)` though this is sacrilegiously not type safe.

Reply:
I'll again argue that stuck expressions should cause an error. They're unusable and unrecoverable in most sane cases (unless you're doing type unsafe things like applying to this `If`)

Reply:
Okay, I 100% agree with you, but stuck expressions are not currently an error and thus this kind of rule can cause a lot of misunderstanding.

Reply:
@developedby  Could not there be a useful case where a stuck expression in a strict function argument would be ignored in a certain rule but instead causes an unwanted error?

We should discuss this in another issue. @o-santi can you remove this change from this PR?

Reply:
Yes, I will do that.